bin/workon.sh

```diff
@@ -37,6 +37,12 @@ pushd ${PROJ_ROOT} >> /dev/null
 # Virtualenv
 # ----------------------------
 
+# disable previous environment with `deactivate` if it exists
```

Comment:
Bash file comments aren't markdown, so we don't need the backticks around `deactivate`. Comments should also start with a capital letter.